tst/EnergyPlus/unit/GroundHeatExchangers.unit.cc

```diff
@@ -1828,7 +1828,20 @@ struct StructBHIntResistanceParameters {
 	}
 };
 
-class GroundHeatExchangerTest_System_calcBHTotalInternalResistance :  public EnergyPlusFixture, public ::testing::WithParamInterface<const StructBHIntResistanceParameters> {};
+class GroundHeatExchangerTest_System_calcBHTotalInternalResistance : public EnergyPlusFixture, public ::testing::WithParamInterface<const StructBHIntResistanceParameters>
```

Comment:
@mbadams5 Do you have any more suggestions on this? I'm still getting errors.

```
'GroundHeatExchangerTest_System_calcBHTotalInternalResistance::SetUpTestCase': cannot access protected member declared in class 'GroundHeatExchangerTest_System_calcBHTotalInternalResistance'
'GroundHeatExchangerTest_System_calcBHTotalInternalResistance::TearDownTestCase': cannot access protected member declared in class 'GroundHeatExchangerTest_System_calcBHTotalInternalResistance'
```
If it is changed to `public` the I get a cannot convert return type error.

`'return': cannot convert from 'testing::internal::ParamGenerator<const T>' to 'testing::internal::ParamGenerator<const StructBHIntResistanceParameters>'`

Thanks